src/js_modules/chooseProgram/index.jsx

```diff
@@ -45,63 +45,71 @@ function ChooseProgram({ chooseList, handleChoose, setLateModalProps }) {
     });
   }) : [];
 
-  const marketingCourses = marketingCursesList?.length > 0 ? marketingCursesList.filter(
+  const hasNonSaasCourse = chooseList.some(({ cohort }) => !cohort.available_as_saas);
+
+  const marketingCourses = marketingCursesList.filter(
```

Comment:
We should compare the cohorts of the student with the cohort of the course. Previously we didn't have the cohort attribute in the course model, now we do